### ISSUE TITLE
Text Contrast failing Accessibility Contrast Checks

### ISSUE THREAD

**Issue by codyseibert** [divanov11/Mumble#155]
There are a lot of places in the application where the contrast is failing accessibility checks which means it can be hard for people with poor eye sight to use the application.  You can check using lighthouse in the browser, or I like to user this snippet sometimes as a bookmark to run https://squizlabs.github.io/HTML_CodeSniffer/.  This issue involves fixing all accessibility issues, but more importantly the contrast ones.

![Screen Shot 2021-04-05 at 1 04 16 AM](https://user-images.githubusercontent.com/1868782/113539169-1dfd0700-95ab-11eb-99b5-0092c61b0336.png)

**Comment by Mr-spShuvo** [comment]
Yeah, right. I would run through test eventually, this just for covering the css variables throughout whole project.

**Comment by Mr-spShuvo** [comment]
Resolve with [this commit ](https://github.com/divanov11/Mumble/commit/9f898654ac41bc037cc4a0c429e86ed0fbc013bf)
